assets/js/interactivity/constants.js

```diff
@@ -0,0 +1,3 @@
+export const cstMetaTagItemprop = 'woo-client-side-transitions';
```

Comment:
I'm not sure if we will remove it later once we switch to Product block-only navigation, but to match the upstream this should now be called `woo-client-side-navigation`.

The [export was also renamed to `csnMetaTagItemprop`](https://github.com/WordPress/block-hydration-experiments/blob/main-wp-directives-plugin/src/runtime/constants.js#L1).

Reply:
OK, I think I've already included those changes in https://github.com/woocommerce/woocommerce-blocks/pull/8447, so I'm going to merge this PR and continue the development there.